base/compiler/ssair/inlining.jl

```diff
@@ -1539,8 +1549,6 @@ function handle_modifyop!_call!(ir::IRCode, idx::Int, stmt::Expr, info::ModifyOp
     edge = info.edges[1]
```

Comment:
@aviatesk is this the right API calls to use to extract the CodeInstance for a result? It is now the only place we call `compileable_specialization` with a MethodInstance instead of a CodeInstance, so it would be nice to get to simplify this callsite too

Reply:
okay, I thought other parts were using `getresult`, so I wasn't sure if that applied here